packages/dashboard/src/components/ShapeContainer/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+
+export const ShapeContainer = styled('div')(({ theme }) => ({
```

Comment:
This name is quite vague. It's originally used as a internal component that does not exported but when act as a public component, we should reconsider this.